static/alert.html

```diff
@@ -208,7 +208,7 @@
                                         If it is pending for longer than the "for" value, it will become a firing alert.'>
                                         </i>
                                     </span>
-                                    <input type="number" name="evaluate-every" id="evaluate-for" value="1" min="0" tabindex="5"
+                                    <input type="number" name="evaluate-every" id="evaluate-for" value="0" min="0" tabindex="5"
```

Comment:
if you use this default value of 0 for `evaluate for` then the alert is not savable, so undo this change

Reply:
done